### ISSUE TITLE
Import Google Code issues

### ISSUE THREAD

**Issue by nemobis** [google/fonts#65]
https://code.google.com/p/googlefontdirectory/issues/ needs to be imported;  there is an import tool and it works. (Via https://phabricator.wikimedia.org/T61983#1384904.)

**Comment by laerm0** [comment]
I have triaged that list into which issues are still useful to import. I am going to get them added in here soon.

**Comment by davelab6** [comment]
@laerm0 using the tool or manually?

**Comment by davelab6** [comment]
I think manually is best as it allows us to de-duplicate, clarify, and prioritize :)

**Comment by laerm0** [comment]
Yep, I planned on doing it manually. As you note, it allows the most control. I'm not doing a bulk import of hundreds of issues so it doesn't need to be automated.
